core/src/main/java/org/opensearch/sql/ast/tree/Window.java

```diff
@@ -24,8 +24,9 @@
 @AllArgsConstructor
 public class Window extends UnresolvedPlan {
 
-  private final UnresolvedExpression windowFunction;
-
+  private final List<UnresolvedExpression> windowFunctionList;
```

Comment:
do we need to add window boundary?  For example, I want to execute a  ppl equal to a sql like 
`select AVG(SUM(total_amount)) OVER (ROWS BETWEEN 2 PRECEDING AND CURRENT ROW) AS moving_avg_3day` (this is one of the `moving avg`, which we plan to support for t2visbuilder).

Reply:
It's easy to support it. but not this PR of `eventstats` command. We can extend this syntax after PPL lang unified.
We can support it when we implement `streamstats` command.

Reply:
I have added a `WindowFrame` for further using.